internal/backend/oauth/oauth.go

```diff
@@ -413,10 +413,6 @@ func (o *oauth) StartFlow(ctx context.Context, intg string, cid sdktypes.Connect
 		return "", errors.New("invalid connection ID")
 	}
 
-	if origin == "" {
```

Comment:
To answer @efiShtain - there's no security aspect

Reply:
do we use it for any security aspect ?